__tests__/app.test.js

```diff
@@ -242,3 +242,103 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("8. PATCH /api/articles/:article_id", () => {
+  test("200: should respond with an updated article", () => {
+    const patchedVotes = { inc_votes: 1 };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(patchedVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
+          author: "butter_bridge",
+          title: "Living in the shadow of a great man",
+          article_id: 1,
+          body: "I find this existence challenging",
+          topic: "mitch",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 101,
+        });
+      });
+  });
+  test("200: should be able to increment article's votes property", () => {
+    const patchedVotes = { inc_votes: 1 };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(patchedVotes)
+      .expect(200)
+      .then(
+        ({
+          body: {
+            article: { votes },
+          },
+        }) => {
+          expect(votes).toBe(101);
```

Comment:
I feel like the test above checked the same thing. Did this fail after you wrote it, then more code was needed?